data/map planets.txt

```diff
@@ -2909,6 +2949,14 @@ planet Parii'het
 	description `Very similar to its sister station of Seli'het, Parii'het was still under construction when it was abandoned by its creators. Today, it floats uninhabited in this rarely visited section of space, approached occasionally by the automated drones that mindlessly continue to carry new supplies to the massive cargo section. If the Ka'het attacked this station, it would be made entirely inoperable in a short time.`
 	government Builder
 
+planet Passaritus
+	attributes "gas giant" "requires: gaslining" uninhabited
+	landscape land/nasa27
+	description `Decades ago, Passaritus was the subject of an ambitious project by the Deep to economically mine metallic hydrogen from its mantle. Being barely large enough to form metallic hydrogen in the first place, the gas giant's gravity was thought to be low enough to make extraction feasible. Unfortunately, Deep engineers overestimated their ability to build an extraction device that was not only technically workable, but economical, and the project has since been moved to the backburner in favor of more reasonable endeavors.`
```

Comment:
```suggestion
	description `Decades ago, Passaritus was the subject of an ambitious project by the Deep to economically mine metallic hydrogen from its mantle. Being barely large enough to form metallic hydrogen in the first place, the gas giant's gravity was thought to be low enough to make extraction feasible. Unfortunately, the Deep's engineers overestimated their ability to build an extraction device that was not only technically workable, but economical, and the project has since been moved to the backburner in favor of more reasonable endeavors.`
```